akka-docs/src/main/paradox/cluster-sharding.md

```diff
@@ -368,6 +381,11 @@ It is always disabled if @ref:[Remembering Entities](#remembering-entities) is e
 <a id="cluster-sharding-remembering"></a>
 ## Remembering Entities
 
+Remembering entities pertains to restarting entities after a rebalance or recovering from a crash.
```

Comment:
Good intro. Perhaps also incorporate something about that they are restarted even though no new messages are sent to them. That is the important difference. Ordinary entities are also started again, but on demand, when a new message arrives.